epoch/api/README.md

```diff
@@ -28,7 +28,26 @@ The epoch node uses the plain HTTP protocol for its API endpoints during the tes
 
 ## WebSocket API definition
 
+### General Request/Response types
+A request has the format:
+
+| Name | Type | Description | Required |
+| ---- | ---- | ----------- | -------- |
+| target | string | what component to target | Yes |
+| action | string | what is the action | Yes |
+| payload | object | data for action | Yes |
+
+A response (and an event) has the format:
+
+| Name | Type | Description | Required |
+| ---- | ---- | ----------- | -------- |
+| origin | string | what component did the action | Yes |
+| action | string | what was the action | Yes |
```

Comment:
I understand this PR documents status quo of WS API, so the following comment is not meant to be addressed in this PR.
This looks like a request-response protocol on top of WS. I wonder what the programming model is: can client send second request before response to first request arrive? If yes, can responses arrive out of order? If yes, do we then need random ref in request for correlating response?

Reply:
A client can send as many requests as one likes and responses will arrive in any order. A desicion had been made that no state changing requests should be made via WS and all requests should be for immutable data (for example getting a block).
We shouldn't be having requestIDs in websockets